lib/supa/command.rb

```diff
@@ -16,8 +18,20 @@ def represent
     private
 
     def value
-      return @representer.send(@options[:modifier], apply_render_flags(raw_value)) if @options[:modifier]
-      apply_render_flags(raw_value)
+      return @value if defined?(@value)
+      value = apply_render_flags(raw_value)
+      @value = modifier ? @representer.send(modifier, value) : value
+    end
+
+    def modifier
+      return @modifier if defined?(@modifier)
+
+      if @options.key?(:modifier) && !@options[:modifier].is_a?(Symbol)
```

Comment:
@Strech Applied your proposition with little change. I added `@options.key?(:modifier)` which is crucial because otherwise it would raise error when the option is not provided.